torch/_torch_docs.py

```diff
@@ -3286,6 +3286,36 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.igamma,
+           r"""
+igamma(input, other, *, out=None) -> Tensor
+
+Regularized lower incomplete gamma function
+
+.. math::
+    \text{out}_{i} = \frac{1}{\Gamma(\text{input})} \int_0^\text{other} t^{\text{input}-1} e^{-t} dt
+
+The shapes of ``input`` and ``other`` must be
```

Comment:
This is good but we're starting to use a new style for statements like this. See the "Supports..." statement here for an example: https://pytorch.org/docs/master/generated/torch.sub.html?highlight=sub#torch.sub.